protocol/rpcprovider/rpcprovider.go

```diff
@@ -488,8 +488,8 @@ func (rpcp *RPCProvider) SetupEndpoint(ctx context.Context, rpcProviderEndpoint
 		utils.LavaFormatTrace("Creating provider node subscription manager", utils.LogAttr("rpcProviderEndpoint", rpcProviderEndpoint))
 		providerNodeSubscriptionManager = chainlib.NewProviderNodeSubscriptionManager(chainRouter, chainParser, rpcProviderServer, rpcp.privKey)
 	}
-
-	rpcProviderServer.ServeRPCRequests(ctx, rpcProviderEndpoint, chainParser, rpcp.rewardServer, providerSessionManager, reliabilityManager, rpcp.privKey, rpcp.cache, chainRouter, rpcp.providerStateTracker, rpcp.addr, rpcp.lavaChainID, DEFAULT_ALLOWED_MISSING_CU, providerMetrics, relaysMonitor, providerNodeSubscriptionManager, rpcp.staticProvider, rpcp.relayLoadLimit)
+	relayLoadProvider := NewProviderLoadManager(rpcp.relayLoadLimit)
```

Comment:
name of variable doesnt match the package. 

providerLoadManager := ...

Reply:
Also, we discussed having a load manager PER chain, this is per API interface. 
we wont get the right load per chain if you initialize it here.

Reply:
Fixed in [725f40a](https://github.com/lavanet/lava/pull/1720/commits/725f40a010e5e88abc7bbdadc1194e0536ab3874)